src/Schema.ts

```diff
@@ -54,11 +54,29 @@ export interface StepWithSelectors extends StepWithFrame {
 export interface ClickStep extends StepWithSelectors {
   type: 'click';
   /**
-   * in px, relative to the top-left corner of the element content box. Defaults to the center of the element
+   * Pointer type for the event. Defaults to 'mouse'.
+   */
+  deviceType?: 'mouse' | 'pen' | 'touch';
+  /**
+   * Defaults to 'primary' if the device type is a mouse.
+   */
+  button?: 'primary' | 'auxiliary' | 'secondary';
+  /**
+   * in ms. Defaults to 50ms.
+   */
+  duration?: number;
+  /**
+   * Defaults to 1.
+   */
+  multiplicity?: 1 | 2;
```

Comment:
what is multiplicity?

Reply:
Double clicks.

Reply:
How would duration work with multiplicity=2?

Reply:
Please include the answer to these into the comments: i.e., multiplicity=2 is double click. `duration` is the time between pointerdown and pointerup and how does it affect double click.

Reply:
These are mutually exclusive. I refrained from writing another even just for double click since it is still a click, and also decided to use multiplicity to mimic Puppeteer. I'm not too opinionated on this however (I chose this style in response to your comments about change/typing events), but if we want to go with Occam's razor, we should just separate the events.

Reply:
We can still document it using union types as the single step type I believe. That will explicitly document exclusivity on the type level.

Reply:
I've thought about this previously, but since our schema here serves as documentation for our SDL, it may lead to confusion. In particular, it can be assumed that each step has a different type, but this would not be true for click and double click since the types are separated implying steps and types are independent. This may set a precedent. we do not want.

Reply:
@OrKoN Check the latest commits to see if there are any improvements.

Reply:
LGTM!